### ISSUE TITLE
Need to copy Tensor to memory

### ISSUE THREAD

**Issue by superlouis** [WilliamCCHuang/GraphLIME#2]
Hey Huang, good work! I was playing around with your repository for my own research and there were 3 tiny issues that I dealt with myself but you may want to fix in your codes.

1. Line 103
2. Line 105
3. Line 157
4. Line 169

Right after detaching the tensor,  add .cpu(). **For example change:**
sample_x = [original_feats.detach().numpy()] to sample_x = [original_feats.detach().cpu().numpy()]

**Comment by WilliamCCHuang** [comment]
@superlouis Thanks for your report. The bugs have been fixed.
